pylintrc

```diff
@@ -0,0 +1,164 @@
+[MASTER]
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=CVS,.git,.cache,.tox,.nox
```

Comment:
You said this was copy-pasted from our project but we definitely don't have `.nox`